_integration-schemas/3plcentral/inventory.md

```diff
@@ -0,0 +1,262 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "inventory"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/inventory.json"
+description: "This table contains information about inventory."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "3PL Central REL documentaion"
```

Comment:
Same as the comment I left for this on the `customers` schema. This should probably be something like `Get inventory`.